### PR TITLE
Converting notebooks for DC2 using the Gen2 Butler

### PR DESCRIPTION
**Pull request by kadrlica**
This is a PR to track the conversion of Stack Club notebooks to DC2 Gen2.

- [x] Basics/ButlerTutorial.ipynb (@DouglasLeeTucker)
- [x] Basics/Calexp_guided_tour.ipynb (@DouglasLeeTucker)
- [x] Basics/Exploring_A_Data_Repo.ipynb (@DouglasLeeTucker)
- [x] ~Basics/Gen3ButlerTutorial.ipynb (@DouglasLeeTucker)~ (not suitable for DC2)
- [x] Basics/afw_table_guided_tour.ipynb  (@DouglasLeeTucker)
- [x] ~Basics/dm_butler_skymap.ipynb~ (never ran on RSP)
- [x] Deblending/scarlet_tutorial.ipynb (@kadrlica)
- [x] Deblending/lsst_stack_deblender.ipynb (@kadrlica)
- [x] GettingStarted/FindingDocs.ipynb (@kadrlica)
- [x] GettingStarted/HelloWorld.ipynb (@kadrlica)
- [x] GettingStarted/ImportTricks.ipynb (@kadrlica)
- [x] ImageProcessing/BrighterFatterCorrection.ipynb (@kadrlica)
- [x] Measurement/UndersampledMoments.ipynb (@kadrlica)
- [x] ~Measurement/DwarfGalaxySrcOverdensity.ipynb~ (not suitable for DC2)
- [x] Measurement/AsteroidLightCurve.ipynb (@MelissaGraham)
- [x] SourceDetection/Footprints.ipynb (@kadrlica)
- [x] SourceDetection/LowSurfaceBrightness.ipynb (@kadrlica)
- [x] Validation/image_quality_demo.ipynb (@DouglasLeeTucker)
- [x] Validation/verify_demo.ipynb (@DouglasLeeTucker)
- [x] Visualization/AFW_Display_Demo.ipynb (@kadrlica)
- [x] Visualization/LsstCameraGeometry.ipynb (@kadrlica)
- [x] Visualization/bokeh_holoviews_datashader.ipynb (@leannep)

### CODE REVIEW

File: Basics/afw_table_guided_tour.ipynb [hunk 1]
```diff
@@ -412,7 +413,62 @@
...
+    "    # Define a dictionary with the filter, ccd, and visit we wish to view\n",
+    "    dataId = {'filter': 'HSC-Z', 'ccd': 32, 'visit': 38938}    \n",
+    "    \n",
+    "    \n",
+    "elif dataset == 'DC2':\n",
+    "\n",
+    "    # The DC2 calexp gen2 repository\n",
+    "\n",
+    "    # The direct path to the file we first want to investigate\n",
+    "    file_path = '/datasets/hsc/repo/rerun/RC/v20_0_0_rc1/DM-25349-sfm/01327/HSC-Z/output/SRC-0038938-032.fits'\n",
```
Comment (kadrlica) [inline Basics/afw_table_guided_tour.ipynb:460]: @DouglasLeeTucker we need to update `file_path` to a path in the DC2 repo.

Comment (kadrlica) [reply Basics/afw_table_guided_tour.ipynb]: I've updated this.

**Comment by kadrlica** [comment]
@DouglasLeeTucker are you willing to sign off on the review for this?

**Comment by DouglasLeeTucker** [comment]
Yes, definitely!


From: Alex Drlica-Wagner ***@***.***>
Date: Friday, April 23, 2021 at 11:53 AM
To: LSSTScienceCollaborations/StackClub ***@***.***>
Cc: Douglas L Tucker ***@***.***>, Mention ***@***.***>
Subject: Re: [LSSTScienceCollaborations/StackClub] Converting notebooks for DC2 using the Gen2 Butler (#252)

@DouglasLeeTucker<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_DouglasLeeTucker&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=R-57Mm7OJX1IswbY4ezVTQ&m=0ASCHzlDnjFjfceZUWbbk8xKfno3hmuIHc7odfD-_Jo&s=09KvOiCx1M8_3vW3GCZMcOy-eT88XkRLB2MM4Ucn7Ls&e=> are you willing to sign off on the review for this?

**Comment by DouglasLeeTucker** [comment]
Looks good!  Let's go with it!

**Review by DouglasLeeTucker — APPROVED** [review]

**Pull request merged**